OSSM-11573 [DOC] Document minimal disruption upgrade steps (#106600)
openshift/openshift-docs: modules/ossm-about-update-strategies-in-ambient-mode.adoc

// Module included in the following assemblies:

// update/ossm-updating-openshift-service-mesh-in-ambient-mode.adoc

:_mod-docs-content-type: CONCEPT
[id="ossm-about-update-strategies-in-ambient-mode_{context}"]
= About the update strategies in ambient mode

[role="_abstract"]

In ambient mode, components update directly through inplace updates. Unlike sidecar mode, ambient mode allows you to move application pods to an upgraded `ztunnel` proxy without restarting or rescheduling the pods.

> Update sequence:: To ensure a successful update in ambient mode, you must follow the following sequence:

Reviewer: ```suggestion
Update sequence:: To update in ambient mode, use the following sequence:
```
